windows/runner/win32_window.cpp

```diff
@@ -173,39 +173,26 @@ Win32Window::MessageHandler(HWND hwnd,
 
       return 0;
     }
-    case WM_SIZE:
+    case WM_SIZE: {
       RECT rect = GetClientArea();
       if (child_content_ != nullptr) {
         // Size and position the child window.
         MoveWindow(child_content_, rect.left, rect.top, rect.right - rect.left,
                    rect.bottom - rect.top, TRUE);
       }
       return 0;
+    }
 
     case WM_ACTIVATE:
       if (child_content_ != nullptr) {
         SetFocus(child_content_);
       }
       return 0;
-    case WM_GETMINMAXINFO:
-      auto info = reinterpret_cast<MINMAXINFO *>(lparam);
-      Size minSize = Size(400, 600);
-      getSizeOnScreen(hwnd, &minSize);
-      info->ptMinTrackSize.x = minSize.width;
-      info->ptMinTrackSize.y = minSize.height;
-      return 0;
```

Comment:
i remember adding this part manually to set the minimum size of the window, idk if it's better to keep it or not tho

Reply:
Better to use bitsdojo window's API which works fine